framework/src/modules/chain/scripts/update_config.js

```diff
@@ -233,6 +251,64 @@ history.version('1.2.0-rc.x', version => {
 	});
 });
 
+history.version('1.6.0-rc.0', version => {
```

Comment:
We have changed the purpose of update_config.js so it stores only the custom modifications of config done by node operators. Hence, a further modification of update_script is not needed here and I foresee the difficulties while releasing.

Reply:
This `update_config.js` is strictly related to our network implementation. So it would be extracted from framework/module and kept on application level. Anyone else create a blockchain from our framework have nothing to do with this config migration, but they still can use our code and can have their own scripts.